codecov/__init__.py

```diff
@@ -134,6 +134,7 @@ def sanitize_arg(replacement, arg):
 
 def write(text, color=None):
     global COLOR
+    text = text or ""
     if COLOR:
```

Comment:
prefer this to be a single `if`

```suggestion
    if text and COLOR:
```

Reply:
I have changed it, but I wonder about the line below:
```
sys.stdout.write(text + "\n")
```